data/arrow.go

```diff
@@ -26,32 +26,10 @@ const metadataKeyRefID = "refId"   // added to the table metadata
 // representation of that table.
 // All fields of a Frame must be of the same length or an error is returned.
 func (f *Frame) MarshalArrow() ([]byte, error) {
-	if _, err := f.RowLen(); err != nil {
-		return nil, err
-	}
-
-	arrowFields, err := buildArrowFields(f)
-	if err != nil {
-		return nil, err
-	}
-
-	schema, err := buildArrowSchema(f, arrowFields)
-	if err != nil {
-		return nil, err
-	}
-
-	columns, err := buildArrowColumns(f, arrowFields)
+	table, err := FrameToArrowTable(f)
 	if err != nil {
 		return nil, err
 	}
-	defer func(cols []arrow.Column) {
-		for _, col := range cols {
-			col.Release()
```

Comment:
I am slightly nervous about missing this release -- however it looks like [table.Release()](https://github.com/apache/arrow/blob/main/go/arrow/array/table.go#L250) will release each column.

I also checked that `mage testRace` works -- and all seems OK

Reply:
So if you use `FrameToArrowTable` it's always recommended/you should call `table.Release()` when done? In that case would be good to clearly state that in the `FrameToArrowTable` description.